packages/devtools_app/lib/src/memory/memory_heap_tree_view.dart

```diff
@@ -528,13 +526,14 @@ class HeapTreeViewState extends State<HeapTree>
                               element.expandCascading();
                               return true;
                             });
-                            setState(() {});
                           }
+                          setState(() {
+                            controller.buildTreeFromAllData();
+                          });
                         }
                       : null,
-                  child: const MaterialIconLabel(
+                  child: const MaterialIconLabel.noText(
```

Comment:
Instead of adding MaterialIconLabel.noText you can just write
`Icon( Icons.vertical_align_bottom, size: 18.0)` or make a helper method with that default. We should probably set 18.0 as our default icon size somewhere.

Reply:
Good idea will create an createIcon helper.